FirebaseCoreDiagnostics.podspec

```diff
@@ -43,9 +44,10 @@ non-Cocoapod integration. This library also respects the Firebase global data co
   }.merge(header_search_paths)
 
   s.source_files = 'Firebase/CoreDiagnostics/FIRCDLibrary/**/*.[cmh]'
+  s.preserve_paths = 'Interop/CoreDiagnostics/Public/*.h'
```

Comment:
Pods that implement an interop protocol should probably have these headers as part of their sources.

Reply:
Agreed - done.